docs/examples/cmark_spec_0.30.yml

```diff
@@ -1740,8 +1818,8 @@ cases:
     mdast:
       type: root
       children:
-        - type: thematicBreak
-        - type: thematicBreak
+        - type: yaml
+          value: ''
     myst: |
       ---
       ---
```

Comment:
This is essentially the only "known" divergence from the commonmark spec, using the "MyST Plugins", i.e. a thematic break on the first line is treated as the front-matter